randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -1220,11 +1246,8 @@ Extra - room_id: [38]
               Can Use Bombs
               # Wave Beam is difficult to control, but can solve the puzzle (Wide Beam could be turned off if also collected)
               Wave Beam and Knowledge (Advanced)
-              All of the following:
-                  # Shinespark directly from adjacent room to the item
-                  Screw Attack and Varia Suit and Shinespark Tricks (Expert) and Disabled Door Lock Rando and Disabled Entrance Rando
-                  # Power Bombs requires Bombs at the moment, but if that changes, Power Bombs alone will not allow the puzzle to be solved in order to leave the room
-                  Can Use Power Bombs
+              # Shinespark directly from adjacent room to the item: https://youtu.be/8Gl3-ZWtDAQ
+              Screw Attack and Varia Suit and Shinespark Tricks (Expert) and Disabled Door Lock Rando and Disabled Entrance Rando and Can Use Power Bombs
```

Comment:
can wave beam also be used instead of pbs to open the door in time?

Reply:
No, a beam shot goes off screen and disappears.